app/src/main/java/org/simple/clinic/drugs/selectionv2/PrescribedDrugEvents.kt

```diff
@@ -7,3 +7,5 @@ import java.util.UUID
 data class ProtocolDrugSelected(val drugName: String, val prescribedDrug: PrescribedDrug?) : UiEvent
 
 data class PrescribedDrugsScreenCreated(val patientUuid: UUID) : UiEvent
+
+data class UpdateCustomPrescription(val prescribedDrug: PrescribedDrug) : UiEvent
```

Comment:
We suffix events by user actions e.g., `Clicked`, `Selected`, `Created`, etc. Can we do the same here? Otherwise, I can't tell when this event gets emitted just by looking at the name.